modules/gdscript/gdscript_utility_functions.cpp

```diff
@@ -250,19 +250,26 @@ struct GDScriptUtilityFunctionsDefinitions {
 		}
 	}
 
-	static inline void load(Variant *r_ret, const Variant **p_args, int p_arg_count, Callable::CallError &r_error) {
+	static inline void load(Variant *r_ret, const Variant **p_args, int p_arg_count, const Ref<GDScript> p_script, Callable::CallError &r_error) {
 		VALIDATE_ARG_COUNT(1);
 		if (p_args[0]->get_type() != Variant::STRING) {
 			r_error.error = Callable::CallError::CALL_ERROR_INVALID_ARGUMENT;
 			r_error.argument = 0;
 			r_error.expected = Variant::STRING;
 			*r_ret = Variant();
 		} else {
+			String path = *p_args[0];
+			if (p_script.is_valid() && p_script->get_path() != "" && !path.is_absolute_path()) {
+				path = p_script->get_path().get_base_dir().path_join(path);
+				path = path.replace("///", "//").simplify_path();
+				Variant *const_override = const_cast<Variant *>(p_args[0]);
+				*const_override = path;
+			}
 			*r_ret = ResourceLoader::load(*p_args[0]);
```

Comment:
Seems like this should be possible as well, but haven't tested.

```suggestion
			}
			*r_ret = ResourceLoader::load(path);
```